virtual_ecosystem/models/hydrology/hydrology_model.py

```diff
@@ -82,6 +81,37 @@ class HydrologyModel(
         "baseflow",
         "bypass_flow",
     ),
+    populated_by_init_vars=(
+        "precipitation_surface",  # precipitation-interception loss
+        "soil_moisture",
+        "vertical_flow",
+        "latent_heat_vapourisation",
+        "molar_density_air",
+        "soil_evaporation",
+        "surface_runoff_accumulated",
+        "subsurface_flow_accumulated",
+        "matric_potential",
+        "groundwater_storage",
+        "river_discharge_rate",
+        "total_river_discharge",
+        "subsurface_flow",
+        "baseflow",
+        "bypass_flow",
+    ),
+    required_update_vars=(
+        "air_temperature",
+        "relative_humidity",
+        "atmospheric_pressure",
+        "precipitation",
+        "wind_speed",
+        "leaf_area_index",
+        "layer_heights",
+        "soil_moisture",
+        "evapotranspiration",
+        "surface_runoff_accumulated",
+        "subsurface_flow_accumulated",
+    ),
+    populated_by_update_vars=(),
 ):
```

Comment:
`air_temperature`, `relative_humididty` and `atmospheric_pressure` are here initilised in a very simple manner under the assumption that the hydrology runs before the abiotic model (which than initializes an actual profile for these variables). Is this still our assumed model order?